data/budget.js

```diff
@@ -4,6 +4,12 @@ const users = require("./users");
 const uuid = require('node-uuid');
 
 let exportedMethods = {
+    
+    getAllBudget() {

```

Comment:
I think this method should be for every user, as a user may have many budgets for eg: budget for categories.